extensions/amp-bind/0.1/validator-amp-bind.protoascii

```diff
@@ -46,15 +46,46 @@ tags: {  # <amp-state> (json)
   }
   spec_url: "https://www.ampproject.org/docs/reference/components/amp-bind"
 }
-tags: {  # <amp-state>
+tags: {  # <amp-state> with json child
   html_format: AMP
   tag_name: "AMP-STATE"
+  spec_name: "amp-state"
   satisfies: "amp-state"
   requires: "amp-bind extension .js script"
   disallowed_ancestor: "AMP-SIDEBAR"
   attrs: {
     name: "id"
     mandatory: true
   }
+  child_tags: {
+    mandatory_num_child_tags: 1
+    first_child_tag_name_oneof: "SCRIPT"
+  }
+  spec_url: "https://www.ampproject.org/docs/reference/components/amp-bind"
+}
+tags: {  # <amp-state> with src
+  html_format: AMP
+  tag_name: "AMP-STATE"
+  spec_name: "amp-state[src]"
+  satisfies: "amp-state"
+  requires: "amp-bind extension .js script"
+  disallowed_ancestor: "AMP-SIDEBAR"
+  attrs: {
+    name: "id"
+    mandatory: true
+  }
+  attrs: { name: "credentials" }
+  attrs: {
+    name: "src"
+    mandatory: true
+    value_url: {
+      allowed_protocol: "https"
+      allow_relative: true  # Will be set to false at a future date.
```

Comment:
Do you want to allow relative URLs?

It not, let's make this false to begin with. If so, remove the comment.

Relative URLs are problematic, because they can be http on the origin.